root/static/scripts/edit/URLCleanup.js

```diff
@@ -1223,6 +1223,28 @@ const CLEANUPS = {
       return url;
     }
   },
+  niconicouser: {
+    match: [new RegExp("^(https?://)?([^/]+\\.)?(nicovideo\\.jp)/user", "i")],
+    type: LINK_TYPES.socialnetwork,
```

Comment:
It should rather fit the relationship type `videochannel` which is a subtype of `socialnetwork`.